pulser-core/pulser/_seq_drawer.py

```diff
@@ -273,77 +298,96 @@ def phase_str(phi: float) -> str:
     t_min = -final_t * 0.03
     t_max = final_t * 1.05
 
-    for ch, (a, b) in ch_axes.items():
+    for ch, axes in ch_axes.items():
         ch_obj = seq._channels[ch]
         basis = ch_obj.basis
         times = np.array(data[ch]["time"])
         t = times / time_scale
-        ya = data[ch]["amp"]
-        yb = data[ch]["detuning"]
+        ys = [data[ch][qty] for qty in ("amp", "detuning", "phase")]
```

Comment:
"amp/detuning/phase" seem to be magic, it would be nice that they be clear from a structure returned by `gather_data`.